### PR TITLE
add SyncErr command for fast checks on current file

### PR DESCRIPTION
**Pull request by dlants**
plus error highlighting!

Add the following to your vimrc:
```viml
" run TSSyncErr on write
autocmd! BufWritePost *.ts,*.tsx TSSyncErr
```

### CODE REVIEW

**Comment by pyrho** [comment]
How about merging this to master? @mhartington

**Comment by mhartington** [comment]
Let me give a test and see how it works

File: rplugin/python3/nvim-typescript/__init__.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ def __init__(self, vim):
         self.files = Dir()
         self._last_input_reload = time()
         self.cwd = os.getcwd()
+        self.src = 0
```
Comment (pyrho) [inline rplugin/python3/nvim-typescript/__init__.py]: If you could just rename this to something more explicit like `self.highlight_source` ?

**Comment by pyrho** [comment]
I've tested it on a relatively large file (~2500 loc) and it does wonders !
Thanks @dlants great stuff.

**Comment by pyrho** [comment]
I've been using it for a week now without issue. Merged.

**Pull request merged**